bin/dev

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+# Find container id of syncm8_api in order to exec things against it
+findBackendContainerId(){
+    local  _resultvar=$1
+    local container=$(docker ps|grep syncm8_api|cut -d' ' -f1)
+    if [ ! $container ]; then
+        echo "No container for api found. Check that it is running with bin/dev ps, or start it with bin/dev up"
+        exit 1
+    fi
+    eval $_resultvar="'$container'"
+}
+
+
+
+# Run docker compose commands as correct user
+runDockerCompose(){
+    user_id=$(id -u) group_id=$(id -g) docker-compose $*
+}
+
+# Create containers
+runDockerUp(){
+    runDockerCompose up --detach
+}
+
+# Create containers and attach backend to shell to faciliatate pdb
+runDockerDebug(){
+    runDockerUp
+    findBackendContainerId container_id
+    cmd="docker attach "$container_id
+    echo $cmd
+    exec $cmd
+}
+
+# Stop and remove all active containers associated with syncm8
+runDockerDown(){
+    runDockerCompose down
+}
+
+# Show running docker images associated with syncm8
+runDockerPs(){
+    docker-compose ps
+}
+
+# Stop and remove all active containers and delete all db data associated with syncm8
+runDockerClean(){
+    runDockerDown
+    docker volume prune -f
+}
+
+# Build images for all syncm8 containers
+runDockerBuild(){
+    runDockerCompose build
+}
+
+# Force rebuild all containers from scratch and update to newest versions
+runDockerRebuild(){
+    runDockerClean
+    runDockerCompose build --pull --no-cache
+}
+
+# Run frontend webpack and attach shell
+runWebpack(){
+    cd client
+    npm run start
+}
+
+# Run pytest
+runPytest(){
+    findBackendContainerId container_id
+    cmd="docker exec "$container_id" pytest "$*
```

Comment:
```suggestion
    cmd="docker exec -t "$container_id" pytest "$*
```
runs pseudo-TTY, it allows colors
https://docs.docker.com/engine/reference/commandline/exec/#options

![image](https://user-images.githubusercontent.com/12789302/139562456-8fc21812-a490-4a37-b697-c3d2cad218d2.png)